airbyte-config/config-models/src/main/java/io/airbyte/config/helpers/StateConverter.java

```diff
@@ -0,0 +1,272 @@
+/*
```

Comment:
@gosusnp @cgardens tagging so you get updates as we continue discussion.

Discussion from parent PR:
![image](https://user-images.githubusercontent.com/7798112/186284212-7ff2b774-c3e1-42f4-a944-1f1f32c31b88.png)

Reply:
@davinchia and I synced about this today, and our plan is to leave StateConverter in the `airbyte-workers` module like Jimmy suggested. `airbyte-server` already depends on `airbyte-workers`, and `airbyte-workers` already needs a dependency on `airbyte-api` to access the client models for creating the API request, so we aren't introducing any new dependencies with this approach.

Reply:
kk